identity/mock-dps-server/src/main.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+#![deny(rust_2018_idioms)]
+#![warn(clippy::all, clippy::pedantic)]
+
+mod request;
+mod server;
+
+#[tokio::main]
+async fn main() {
+    let matches = clap::App::new("mock-dps-server")
```

Comment:
Use structopt, like we do for the other binaries.

Reply:
Fixed